src/Exception/EntityNotFoundException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Linio\Common\Exception;
+
+use Throwable;
+
+class EntityNotFoundException extends ClientException
+{
+    public function __construct(
+        string $token = ExceptionTokens::ENTITY_NOT_FOUND,
+        string $message = ExceptionTokens::ENTITY_NOT_FOUND,
```

Comment:
The big downside though of not using `$message` or `$internalDetail` to send to `\Exception`'s `$message` parameter is that handlers that are not aware of our exceptions will not have the additional details. For example the NewRelic listener will not send to NewRelic the full details because it's not aware of them.

I think we have to stick with `$message` as it is implemented currently. We can't go overriding everything in every application.